config/FindGrace.cmake

```diff
@@ -6,77 +6,201 @@
 # note   Copyright (C) 2016-2017 Los Alamos National Security, LLC.
 #        All rights reserved.
 #------------------------------------------------------------------------------#
-# - Find Grace
-# Find the native Grace includes and library
+
+# FindGrace
+# ---------
+#
+# Find the native Grace binary, includes and library.
+#
+# Grace is a WYSIWYG 2D plotting tool for the X Window System and M*tif. Grace
+# runs on practically any version of Unix-like OS. As well, it has been
+# successfully ported to VMS, OS/2, and Win9*/NT/2000/XP (some minor
+# functionality may be missing, though). Grace is a descendant of ACE/gr, also
+# known as Xmgr.
+# http://plasma-gate.weizmann.ac.il/Grace/
+#
+# Imported Targets
+# ^^^^^^^^^^^^^^^^
+#
+# If Grace is found, this module defines the following :prop_tgt:`IMPORTED`
+# targets::
 #
+# Grace::grace          - The grace library.
+#
+# Result Variables
+# ^^^^^^^^^^^^^^^^
+#
+# This module will set the following variables in your project::
+#
+#  Grace_FOUND          - True if Grace found.
 #  Grace_INCLUDE_DIRS   - where to find Grace.h, etc.
 #  Grace_LIBRARIES      - List of libraries when using Grace.
-#  GRACE_FOUND          - True if Grace found.
+#  Grace_EXECUTABLE     - main program
+#
+# Hints
+# ^^^^^
+#
+# Set ``Grace_ROOT_DIR`` to a directory that contains a Grace installation.
```

Comment:
I can make this change.  I used `Grace_ROOT_DIR` because some other TPL used that naming convention.  It looks like Grace uses `GRACE_HOME` in their documentation, so I will switch it.